grbl/grbl.h

```diff
@@ -22,10 +22,10 @@
 #define grbl_h
 
 // Grbl versioning system
-#define GRBL_VERSION "1.1h-XCP"
+#define GRBL_VERSION "1.1h-XCPd"
```

Comment:
The ready response string was updated to match what Easel is recognizing. I used `d` for `door` with the changes in pin behavior, but I'm open to other naming schemes if we decide to start versioning firmware in more organized way.

Reply:
An alternative would be to instead trigger different behavior based on the `$I` output which has a bunch of different flags for which features are enabled. Potentially we encode this information there rather than trying to overload this all based on the version number